docs/docs/contributor/local-setup/yarn-setup.mdx

```diff
@@ -9,10 +9,10 @@ sidebar_custom_props:
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-In this document, you'll learn how to install the project using yarn. We recommend this method since it's the easiest way to get started but you can also run the project with [Docker](/contributor/local-setup/docker-setup).  
+In this document, you'll learn how to install the project using yarn. You should use this method since it's the easiest way to get started but you can also run the project with [Docker](/contributor/local-setup/docker-setup).  
 
 :::info
-`npm` currently does not support local packages satisfactorily. We strongly recommend using `yarn` instead.
+Right now `npm` does not support local packages satisfactorily. That's why it's better to use `yarn` instead.
```

Comment:
Remove "Right now", the goal is to simplify not find synonyms
Also "That's why it's better" feels like an overall judgment that yarn is better 
E.g. "Npm does not support local packages well, opt for `yarn` instead"